### PR TITLE
Update to Pyodide 0.22

### LINKED ISSUE #919: Loading vanilla Seaborn dataset errors out: `urlopen error unknown url type: https`
## Description

I was going through the [seaborn tutorial](https://seaborn.pydata.org/tutorial/introduction.html) and JupyterLite failed at the first example because it couldn't load the dataset due to something to do with https. The error is that it uses an https protocol. This sort of reminds me of [#744] but I don't see a similar fix.

## Reproduce

```python
%pip install seaborn

import seaborn as sns

# Load an example dataset
tips = sns.load_dataset("tips")
```

## Output

```
---------------------------------------------------------------------------
URLError                                  Traceback (most recent call last)
Cell In[1], line 6
      3 import seaborn as sns
      5 # Load an example dataset
----> 6 tips = sns.load_dataset("tips")

File /lib/python3.10/site-packages/seaborn/utils.py:582, in load_dataset(name, cache, data_home, **kws)
    580 cache_path = os.path.join(get_data_home(data_home), os.path.basename(url))
    581 if not os.path.exists(cache_path):
--> 582     if name not in get_dataset_names():
    583         raise ValueError(f"'{name}' is not one of the example datasets.")
    584     urlretrieve(url, cache_path)

File /lib/python3.10/site-packages/seaborn/utils.py:508, in get_dataset_names()
    502 """Report available example datasets, useful for reporting issues.
    503 
    504 Requires an internet connection.
    505 
    506 """
    507 url = "https://github.com/mwaskom/seaborn-data"
--> 508 with urlopen(url) as resp:
    509     html = resp.read()
    511 pat = r"/mwaskom/seaborn-data/blob/master/(\w*).csv"

File /lib/python3.10/urllib/request.py:216, in urlopen(url, data, timeout, cafile, capath, cadefault, context)
    214 else:
    215     opener = _opener
--> 216 return opener.open(url, data, timeout)

File /lib/python3.10/urllib/request.py:519, in OpenerDirector.open(self, fullurl, data, timeout)
    516     req = meth(req)
    518 sys.audit('urllib.Request', req.full_url, req.data, req.headers, req.get_method())
--> 519 response = self._open(req, data)
    521 # post-process response
    522 meth_name = protocol+"_response"

File /lib/python3.10/urllib/request.py:541, in OpenerDirector._open(self, req, data)
    538 if result:
    539     return result
--> 541 return self._call_chain(self.handle_open, 'unknown',
    542                         'unknown_open', req)

File /lib/python3.10/urllib/request.py:496, in OpenerDirector._call_chain(self, chain, kind, meth_name, *args)
    494 for handler in handlers:
    495     func = getattr(handler, meth_name)
--> 496     result = func(*args)
    497     if result is not None:
    498         return result

File /lib/python3.10/urllib/request.py:1419, in UnknownHandler.unknown_open(self, req)
   1417 def unknown_open(self, req):
   1418     type = req.type
-> 1419     raise URLError('unknown url type: %s' % type)

URLError: <urlopen error unknown url type: https>
```

## Context

JupyterLite Version 0.1.0b15
Chrome Browser Version 108.0.5359.98 (Official Build) (arm64)

### PR DESCRIPTION
**Pull request by jtpio**
## References

Fixes https://github.com/jupyterlite/jupyterlite/issues/909
Closes https://github.com/jupyterlite/jupyterlite/pull/930

## Code changes

Address items listed in #909:

- [x] Update the Pyodide version
- [x] Install `sqlite3` at kernel startup
- [x] Update the name of the distribution

Also update docs to:

- [x] Document fetching remote content, since `pyodide-http` is now part of the Pyodide distribution: https://github.com/pyodide/pyodide/pull/3355

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by jtpio** [comment]
Doing the update here instead of https://github.com/jupyterlite/pyodide-kernel since Pyodide 0.22 final is about to be released.

File: packages/pyolite-kernel/package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "@jupyterlab/testutils": "~3.5.2",
     "@types/jest": "^26.0.10",
     "jest": "^26.4.2",
-    "pyodide": "0.21.3",
+    "pyodide": "0.22.0-alpha.3",
```
Comment (bollwyvl) [inline packages/pyolite-kernel/package.json:61]: needs bump, and re-lock

File: docs/howto/content/python.md [hunk 2]
```diff
@@ -39,3 +39,39 @@ os.getcwd()  # If successful:  "/drive/path/to/notebook"
   https://github.com/jupyterlite/jupyterlite/blob/main/examples/pyolite/emscripten-filesystem.ipynb
 [fs]: https://emscripten.org/docs/api_reference/Filesystem-API.html
 [caniuse-sw]: https://caniuse.com/serviceworkers
+
+## Fetching remote content
```
Comment (bollwyvl) [inline docs/howto/content/python.md:43]: While this is good, perhaps actually having a demo notebook (maybe showing seaborn) would be useful as well... and to ensure it _keeps_ working.

**Review by bollwyvl — APPROVED** [review]
LGTM, `%` that maybe we want an interactive example of using `pyodide_http`

File: docs/howto/content/python.md (hunk 2 above)
Comment (jtpio) [reply docs/howto/content/python.md:43]: Good point, thanks :+1:

**Comment by jtpio** [comment]
> LGTM, `%` that maybe we want an interactive example of using `pyodide_http`

Sounds good, I'll open a follow-up PR to address this, which will fix https://github.com/jupyterlite/jupyterlite/issues/919.

**Comment by jtpio** [comment]
Thanks for the review!

**Pull request merged**